### ISSUE TITLE
[Project Plan] Fill the Project Plan Covering the Activities until Milestone 1

### ISSUE THREAD

**Issue by CahidArda** [bounswe/bounswe2022group7#133]
A Project Libre project will be created and the activities covering the CMPE352 course until the Miletstone 1 will be added.

In order to identify the activities and the dates, meeting notes and the RAM will be used as reference.

Deadline: 15/04/2022, 14:00
Reviewer: @canatakan

**Comment by CahidArda** [comment]
The project plan [document.pdf](https://github.com/bounswe/bounswe2022group7/files/8495080/cmpe352.pdf) covering the activities until the end of milestone 1 is created. Also the Project Libre project file is shared with teammates.

Ready for review @canatakan!

**Comment by canatakan** [comment]
I have used the relevant Project Libre document to proceed with my work. The plan was well-written in general. Here is a few significant changes that I did while completing the Project Plan:

* I used `Predecessor` feature to have a sequential relationship between the tasks
* I added team members' names to `Resources` part so that it is possible to assign the events
* I deleted the events related to API since I included those under  the "Practice App" headline, with a much more detailed planning

The issue about completion of the plan can be followed at #129 from now on. Thank you for your contribution.
